index.hbs

```diff
@@ -76,11 +76,85 @@ into the {body} of the default.hbs template --}}
             {{/if}}
           {{/get}}
           {{!-- The tag below iterates over all the posts --}}
-          {{> "loop"}}
-        </div>
-      </div>
+          {{!-- {{> "loop"}} 
+        </div> 
+        </div> --}}
+
+        <div class="l-wrapper in-post aos-init aos-animate" data-aos="fade-up" data-aos-delay="300">
+            <div class="l-post-content">
+              <div class="pos-relative js-post-content">
+                <h1 class="ieee-blue">Who We Are</h1>
+                <h3 class="ieee-blue">Student Leaders</h2>
+                <div class="paragraph-div">
+                  <p class="paragraph">
+                    The University of California, Berkeley IEEE Student Branch was established in 1912 and has been one of the most active engineering student organizations on campus ever since.
+                  </p>
+                </div>
+
+                <h3 class="ieee-blue">What We Do</h1>
+                <div class="paragraph-div">
+                  <p class="paragraph">
+                    We aim to promote professional development, encourage academic growth, and organize social activities within the greater engineering community. <br />
+                    Some of the activities we host:
+                    <div class="row">
+                      <div class="two-columns">
+                        <ul class="paragraph">
+                          <li>Technical Workshops</li>
+                          <li>Resume and Interview Workshops</li>
+                          <li>Startup Fair</li>
+                          <li>Dinner with Corporate Sponsors</li>
+                        </ul>
+                      </div>
+                      <div class="two-columns">
+                        <ul class="paragraph">
+                          <li>Company Info-sessions</li>
+                          <li>Mentorship Program</li>
+                          <li>Two Decals (Hope, Micromouse)</li>
+                          <li>Socials (Dinner, bowling, ice-skating, etc.)</li>
+                        </ul>
+                      </div>
+                    </div>
+                  </p>
+                </div>  {{!-- End of paragraph-div --}}
+
+
+                <div class="stats-div text-center div-spacing">
+                  <div class="row">
+                    <div class="three-columns stats-columns">
+                      <div class="stats-bubbles">
+                        <span class="stats">500 General Members</span>
+                      </div>
```

Comment:
The ellipses should be a circle:
```html
<div class="circle-text">
    <p>500</p>
    <p>General</p>
    <p>Members</p>
<div>
```

```scss
.circle-text {
  padding: 30px;
  border-radius: 3000px;
  text-align: center;
  background-color: blue;
  display: inline-block; // This can be ignored if you use a container that has "display: flex;"
  color: white;
  p {
    margin: 0;
  }
}
```